sourcegraph/learn
# Linting updates

### What should this PR do?
Resolves [DEVED-287](https://sourcegraph.atlassian.net/browse/DEVED-287) by updating our dependencies to get linting working again.

### Why are we making this change?
- We need to revert #403  for the reasons explained in #361. We also need to install some more things required by `sourcegraph/eslint-config`

### What are the acceptance criteria? 
- Running `npm run lint` should work.

### How should this PR be tested?
- Check out the branch, remove `node_modules` and `npm ci`. Then, run the above command.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
